config/default

```diff
@@ -60,7 +60,7 @@ export mail_debug = 0
 
 # FIXME
 export discourse_sso_secret = d836444a9e4084d5b224a60c208dce14
-export discourse_api_key = 4647c0d98e8beb793da099ff103b9793d8d4f94fff7cdd52d58391c6fa025845
+export discourse_api_key = c1706802dbf1152c91492a9dbd5dc6e1480a1de57c25089401b3f2416d8f6a1a
```

Comment:
@gberaudo has reverted to the first key last week. See https://github.com/c2corg/v6_ui/issues/452#issuecomment-228051981
I don't know why the key had changed in the meantime.